packages/components/icon/src/icon.ts

```diff
@@ -5,6 +5,7 @@ export interface IconProps {
   nodeType?: string
   fontSize?: string | number
   color?: string
+  isDisabled?: boolean
```

Comment:
Is it belongs to this PR ? It seems that it's a commit by @MikeLebeau

Reply:
oui c'est suite au merge de "Merge branch 'feat/2.0.0-beta'"